local/rest_api_gcbm/app.py

```diff
@@ -342,6 +342,8 @@ def gcbm_download():
     # Sanitize title
     title = "".join(c for c in title if c.isalnum())
     project_dir = f"{title}"
+    if not os.path.exists(f"{os.getcwd()}/input/{project_dir}"):
+        return {"error": "No such simulation exist for this {title}"}, 400
```

Comment:
I think the error status code, can be changed to `404` as `400` is used for a bad request and `404` is for a not found error. Let me know your thoughts on this.

Reply:
I think 404 is valid here because the simulation of the given title is not present. @padmajabhol What are your views on this issue.